README.md

```diff
@@ -1,5 +1,5 @@
 # The gridscale Go Client
-[![Build Status](https://travis-ci.com/gridscale/gsclient-go.svg?branch=master)](https://travis-ci.com/gridscale/gsclient-go) [![Go Report Card](https://goreportcard.com/badge/github.com/gridscale/gsclient-go)](https://goreportcard.com/report/github.com/gridscale/gsclient-go)
+[![Build Status](https://travis-ci.com/gridscale/gsclient-go.svg?branch=develop)](https://travis-ci.com/gridscale/gsclient-go) [![Go Report Card](https://goreportcard.com/badge/github.com/gridscale/gsclient-go)](https://goreportcard.com/report/github.com/gridscale/gsclient-go)
```

Comment:
I would suggest keeping the master branch because at the end it shows the status of what merged to the master branch